core/src/banking_stage.rs

```diff
@@ -3232,6 +3236,10 @@ mod tests {
             ..
         } = create_slow_genesis_config(10_000);
         let bank = Arc::new(Bank::new_no_wallclock_throttle_for_tests(&genesis_config));
+        // set cost tracker limits to MAX so it will not filter out TXs
+        bank.write_cost_tracker()
+            .unwrap()
+            .set_limits(std::u64::MAX, std::u64::MAX, std::u64::MAX);
```

Comment:
maybe we can just fold this into `Bank::new_no_wallclock_throttle_for_tests` and rename that function to ``Bank::unthrottled_for_tests`